tests/state_test_util.go

```diff
@@ -366,6 +371,13 @@ func useEthGasPrice(config *params.ChainConfig, json *stJSON) (*big.Int, error)
 	return gasPrice, nil
 }
 
+func useEthOpCodeGas(r params.Rules, evm *vm.EVM) {
+	if r.IsCancun {
+		// EIP-1052 must be retained for backward compatibility.
+		vm.ChangeGasCostForTest(&evm.Config.JumpTable, vm.EXTCODEHASH, params.WarmStorageReadCostEIP2929)
+	}
+}
+
```

Comment:
Could you explain how it is introduced?

Reply:
https://github.com/kaiachain/kaia/pull/129#discussion_r1841399589

This is related to this comment. The constant gas of EXTCODEHASH is different because EIP-1052 is never activated on Ethereum. The relevant tests were being skipped, but I fixed it so that they can be run with a constant gas change.

Reply:
Thanks for the information.

> // EIP-1052 must be retained for backward compatibility.

Then, the comment seems like it needs to be clarified. What do you think about this?
`// EIP-1052 must be inactivated for backward compatibility.`

Reply:
I'll add the details.

Reply:
I fixed this at https://github.com/kaiachain/kaia/pull/129/commits/0afb6440cf1ae0c894ae10398a31cfa61b0903a7.